string station_name
int32 station_id
string opDescriptor
